### ISSUE TITLE
searching "mission impossible" in movies script

### ISSUE THREAD

**Issue by hallzy** [alexanderepstein/Bash-Snippets#29]
**Issue Label:**
* [x] bug
* [ ] feature
* [ ] enhancement

**Description of bug or feature enhancement:**

When I run the movies script to find "mission impossible" like so, I get this output:

```bash
$ ./movies/movies mission impossible
Traceback (most recent call last):
  File "<string>", line 1, in <module>
IndexError: list index out of range

==================================================
| Title: Mission Impossible
| Year: 2008
| Tomato: 
| Rated: N/A
| Genre: Short, Action
| Director: Travis Neal
| Actors: Travis Neal
| Plot: N/A
==================================================
```

Ignoring the fact that this is not the movie I was expecting (though, you can get the tom cruise movie with "mission: impossible" this execution produces an index error.

The problem seems to come from line 58 where the code tries to find the Tomato score, sine this output seems to not have one.

One possible solution is to just pipe the error results of the command to /dev/null as below.

```bash
score=$(echo $movieInfo | python2 -c "import sys, json; print json.load(sys.stdin)['Ratings'][1]['Value']" 2> /dev/null)
```

I have tested that line and the error does in fact go away. If this solution works for you then let me know and I can create a PR for it, or you can make the fix yourself with this method or another method you may have in mind.

**OS and OS version:**
* [ ] Mac
* [ ] Linux 32 Bit
* [x] Linux 64 Bit
* [ ] Windows 32 Bit
* [ ] Windows 64 Bit

OS Version: 16.04LTS

**Comment by alexanderepstein** [comment]
Will add a similar fix later I am going to check if the variable has been set to anything, if not I don't want ```Tomato:``` showing up in the ouput

**Comment by alexanderepstein** [comment]
Fixed 69466559dce7a273c7c0d24461cbc868251741db
